frontend/src/lib/components/CreateServiceRequest/DisplayServiceDefinitionAttributes/DisplayMultiAttribute.svelte

```diff
@@ -7,13 +7,19 @@
 		let values: string[] = [];
 
 		const keys = attributes.value;
-		for (let attribute of attributes.attribute.values) {
-			if (keys && keys.includes(attribute.key)) {
-				values.push(attribute.name);
+		if (attributes.attribute.values) {
```

Comment:
I don't think these changes should be included because they are guarding against an invalid server state.   The backend is what should be fixed to address the bug.